configuration/renci_testbed/renci_ben_backup_port.manifest

```diff
@@ -0,0 +1,400 @@
+      "type": "dtn",
+      "friendlyname": "RENCI BF40G Node",
+      "location": "35.939796, -79.018203",
+      "vlan": 1421},
+    "rencibm8": {
+      "type": "dtn",
+      "friendlyname": "RENCI BM8 Node",
+      "location": "35.939796, -79.018203",
+      "vlan": 1425},
+    "dukectlr": {
+      "type": "localcontroller",
+      "friendlyname": "DUKE Controller",
+      "location": "36.003614, -78.938755",
+      "vlan": 1411},
+    "dukedtn": {
+      "type": "dtn",
+      "friendlyname": "DUKE BF40G Node",
+      "location": "36.003614, -78.938755",
+      "vlan": 1422},
+    "uncctlr": {
+      "type": "localcontroller",
+      "friendlyname": "UNC Controller",
+      "location": "35.903395, -79.048370",
+      "vlan": 1411},
+    "uncdtn": {
+      "type": "dtn",
+      "friendlyname": "UNC BF40G Node",
+      "location": "35.903395, -79.048370",
+      "vlan": 1423},
+    "ncsuctlr": {
+      "type": "localcontroller",
+      "friendlyname": "NCSU Controller",
+      "location": "35.773395, -78.677730",
+      "vlan": 1411},
+    "ncsudtn": {
+      "type": "dtn",
+      "friendlyname": "NCSU BF40G Node",
+      "location": "35.773395, -78.677730",
+      "vlan": 1424}
+  },
+  "localcontrollers": {
+    "rencictlr":{
+      "shortname": "rencictlr",
+      "credentials": "rencipwd",
+      "location": "35.939796, -79.018203",
+      "lcip": "10.14.11.1",
+      "internalconfig": {
+        "ryucxninternalport": 55781,
+        "openflowport": 6681,
+        "backuplcswitch": "dukes1"
+      },
+      "switchinfo": [
+        {
+          "name":"rencis1",
+          "friendlyname":"RENCI Corsa DP2100",
+          "ip": "10.0.201.1",
+          "dpid":"201",
+          "brand":"Open vSwitch",
+          "model": "2.3.0",
+          "portinfo": [
+            {
+      	      "portnumber": 1,
+              "speed": 80000000000,
+              "destination": "uncs1"
+	    },
+            {
+              "portnumber": 2,
+              "speed": 80000000000,
+              "destination": "dukes1"
+            },
+            { 
+              "portnumber": 23,
+              "speed": 80000000000,
+              "destination": "rencis2"
+            },
+            {
+      	      "portnumber": 11,
+	       "speed": 80000000000,
+	       "destination": "rencictlr"
+	    },
+            {
+      	      "portnumber": 12,
+	       "speed": 80000000000,
+	       "destination": "rencidtn"
+	    },
+            {
+      	      "portnumber": 30,
+	       "speed": 80000000000,
+	       "destination": "sdx"
+	    }
+	  ],
+          "internalconfig": {
+            "corsaurl": "https://corsa-2.renci.ben/",
+            "corsatoken": "ddc60638fa7a7aa3fb5ca10de8f4e5e8bf82cd289187f933cfc7d7a01e7f7f3839ecac1145bc9908abfd03aa493e4acda448522b304a6ce779f82ce9f1528356",
+            "corsabridge": "br21",
+            "corsabwin": 19,
+            "corsabwout": 20,
+            "corsaratelimitbridge": "br20",
+            "corsaratelimitports": [21,22],
```

Comment:
Since this branch is a new feature added on top of current functionalities, 
it would be ideal to include the l2mp keys to this manifest. (`corsabwinl2mp, corsabwoutl2mp, corsaratelimitbridgel2mp, corsaratelimitportsl2mp`)

https://github.com/atlanticwave-sdx/atlanticwave-proto/blob/061c44632229fe8a88bba502353febaef061b1a2/configuration/renci_testbed/renci_ben.manifest#L112

Also, file seems to have some indentation issues at a few places.